.gitignore

```diff
@@ -30,8 +30,12 @@ yarn-error.log*
 /kumascript/coverage/
 /kumascript/node_modules/
 /testing/node_modules/
-/client/build/
+/client/build
+/client/src/**/*.js
+/client/src/**/*.js.map
```

Comment:
@caugner where and how would these be created?

Reply:
`tool/cli.ts` imports from `client/src/document/types.ts`, so tsc will transpile `client/src/document/types.ts` as well.

Reply:
^ @fiji-flo